pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -155,10 +155,18 @@ func Run(f *clientcmd.Factory, options *ipfailover.IPFailoverConfigCmdOptions, c
 		return err
 	}
 
-	if len(options.ServiceAccount) == 0 {
-		return fmt.Errorf("you must specify a service account for the ipfailover pod with --service-account, it cannot be blank")
+	if options.VRRPIDOffset < 0 || options.VRRPIDOffset > 255 {
+		return fmt.Errorf("The vrrp-id-offset must be in the range 0..255")
```

Comment:
I don't see that.  In config-generators.sh line 270 it creates a counter variable that starts at 1.  That is passed into generate_vrrpd_instance_config at line 290 and is put into $iid (although the default of 0 is probably wrong and we probably should change the $iid default at line 209 to 1).

Then we add that to the offset.  So the first allocated will be offset + 1.  So the range you can set the offset to only goes to 254.